package.json

```diff
@@ -100,6 +104,7 @@
     "karma-sourcemap-loader": "^0.3.5",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.3.3",
+    "nightwatch": "^0.9.5",
     "phantomjs": "^1.9.18",
     "phantomjs-polyfill": "0.0.1",
     "react-hot-loader": "^1.3.0",
```

Comment:
_NOTE: npm likes things in alphabetical order hence, the exaggerated diff here_

Packages modified (most of these are already meged in #64):
+ +`nightwatch`
+ +`glob`
+ +`commander`
+ +`colors`
+ +`babel-preset-react`
+ +`babel-preset-stage-0`
+ +`babel-preset-es2015`
+ +`babel-plugin-transform-decrators-legacy`
+ -`babel` (not needed in babel 6)
+ -`babel-runtime` (not needed in babel 6)
+ -`babel-plugin-react-transform` (replaced by `babel-preset-react`)